guides/v2.1/frontend-dev-guide/themes/theme-apply.md

```diff
@@ -56,7 +56,7 @@ To add a permanent theme-independent logo:
 2. In the configuration record corresponding to your store view, click **Edit**.
 3. Expand the **Header** tab.
 4. In the **Logo Image** field browse to the logo file saved in your file system.
-6. Upload the file.
+6. Upload the file. Allowed file types include png, gif, jpg, and jpeg. To add an .svg logo, see [Declaring theme logo]({{page.baseurl}}/frontend-dev-guide/themes/theme-create.html#logo_declare).
```

Comment:
```suggestion
6. Upload the file. Allowed file types include .png, .gif, .jpg, and .jpeg. To add an .svg logo, see [Declaring theme logo]({{page.baseurl}}/frontend-dev-guide/themes/theme-create.html#logo_declare).
```

Reply:
Sorry @abrarpathan19, I didn't catch this in my initial review. Make these small updates and I'll merge your PR.